HW5/TanchakYaroslav/loop_task.py

```diff
@@ -0,0 +1,45 @@
+# Task 1
+# list_numbers = [23, 45, 71, 12, 516, 3, 57, 10]
+#
+# for i in range(len(list_numbers)):
+#     list_numbers[i] = float(list_numbers[i])
+
+# Task 2
+
+# Version 1
+# fibonacci_list = [0, 1]
+# numb_fibonacci = int(input("Enter number for sequence of Fibonacci, please - "))
+#
+# for i in range(abs(numb_fibonacci)):
+#     next_fib_numb = fibonacci_list[-2] + fibonacci_list[-1]
+#     if next_fib_numb > numb_fibonacci:
+#         break
+#     fibonacci_list.append(next_fib_numb)
+#
+# print(fibonacci_list)
+
```

Comment:
excellent work
look at this implementation
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])
print(f'there are some numbers from fibanacci till that you entered {n},\n {fibonacci_numbers}')